ios/FluentUI/Vnext/Button/Button.swift

```diff
@@ -118,33 +103,33 @@ class MSFButtonStateImpl: NSObject, ObservableObject, MSFButtonState {
 /// Body of the button adjusted for pressed or rest state
 struct FluentButtonBody: View {
     @Environment(\.isEnabled) var isEnabled: Bool
-    @ObservedObject var tokens: MSFButtonTokens
     @ObservedObject var state: MSFButtonStateImpl
+    var tokens: ButtonTokens
     let isPressed: Bool
 
     var body: some View {
         let isDisabled = !isEnabled
         let isFloatingStyle = tokens.style.isFloatingStyle
         let shouldUsePressedShadow = isDisabled || isPressed
-        let iconColor: UIColor
-        let titleColor: UIColor
-        let borderColor: UIColor
-        let backgroundColor: UIColor
+        let iconColor: DynamicColor
+        let titleColor: DynamicColor
+        let borderColor: DynamicColor
+        let backgroundColor: DynamicColor
         if isDisabled {
-            iconColor = tokens.disabledIconColor
-            titleColor = tokens.disabledTitleColor
-            borderColor = tokens.disabledBorderColor
-            backgroundColor = tokens.disabledBackgroundColor
+            iconColor = tokens.iconColor.disabled
+            titleColor = tokens.textColor.disabled
```

Comment:
Should we change titleColor to textColor? It's the only one that doesn't match its token.